tool/jt.rb

```diff
@@ -2002,6 +2003,75 @@ def profile(*args)
     app_open svg_filename
   end
 
+  def graph(*args)
+    test_file = nil
+    method = 'Object#foo'
```

Comment:
Do you mean to hardcode the method foo? I thought user can specify the method e.g. `jt graph --method fib test.rb`

Reply:
This is a default - it's overwritten below.